test/PublicApiTests/BaseLine/Microsoft.OData.PublicApi.net45.bsl

```diff
@@ -7925,7 +7925,6 @@ public enum Microsoft.OData.Client.SaveChangesOptions : int {
     BatchWithSingleChangeset = 1
     BulkUpdate = 128
     ContinueOnError = 2
-    DeepInsert = 256

```

Comment:
Undo this change since it is unrelated to this PR

Reply:
One of previous commit forgot to update the bsl.